pkg/app/pipedv1/plugin/azure/config/func_config.go

```diff
@@ -0,0 +1,22 @@
+package config
+
+type AzureDeployTemplate struct {
+	DeploymentName          string `json:"deploymentName"`
+	DeploymentTemplateFile  string `json:"deploymentTemplateFile"`
+	DeploymentParameterFile string `json:"deploymentParameterFile"`
+}
+type FunctionsSpec struct {
+	FunctionName      string               `json:"functionName"`
+	ResourceGroupName string               `json:"resourceGroupName"`
+	ArmTemplate       *AzureDeployTemplate `json:"armTemplate"`
+	BicepTemplate     *AzureDeployTemplate `json:"bicepTemplate"`
+	PackageUri        string               `json:"packageUri"`
+}
+
+type AzureFunctionSyncStageConfig struct {
+	SlotName string `json:"slot"`
+}
+type AzureFunctionSwapStageConfig struct {
+	SlotName1 string `json:"slot1"`
+	SlotName2 string `json:"slot2"`
```

Comment:
Name1&2 are not clear to users and developers

```suggestion
	PrimarySlot string `json:"primarySlot"`
	CanarySlot string `json:"canarySlot"`
```